doc/module_examples/02_inspection_needed/check.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+. /usr/share/preupgrade/common.sh
+
+#END GENERATED SECTION
+
+#
+# The whole section above is processed and modified by the preupg-xccdf-compose
+# script, according to the contents of the INI file (in this case it is the
+# content.ini file). In addition, the LICENSE used by the Preupgrade Assistant
+# is inserted.
+#
+
+##
+# Briefly:
```

Comment:
I disagree. IMO it makes whole module much clear just from this piece of text. It whould be ideally inside each module >= 02. The reader will be able to skip it if it is not interested about this text and got down to the code.